stable/locust/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: locust
 description: A modern load testing framework
-version: 0.1.4
+version: 0.1.5
```

Comment:
Please bump the minor version since you added a new feature.

Reply:
:+1: